pkg/kuberneteshelper/pod.go

```diff
@@ -105,3 +108,32 @@ func GetOsmConfigurator(osmNamespace common.MeshNamespace) configurator.Configur
 	cfg := configurator.NewConfigurator(versioned.NewForConfigOrDie(kubeConfig), stop, osmNamespace.String(), constants.OSMMeshConfig)
 	return cfg
 }
+
+// GetMatchingServices returns a list of Kuberentes services in the namespace that match the pod's label
+func GetMatchingServices(kubeClient kubernetes.Interface, podLabels map[string]string, namespace string) ([]service.MeshService, error) {
+	serviceList, err := kubeClient.CoreV1().Services(namespace).List(context.Background(), v12.ListOptions{})
```

Comment:
Perhaps you can rename the imports from `v1` and `v12` to `corev1` and `metav1` (above). The `v12` import is confusing especially here.

Reply:
yeah tbh I'm thinking we need to do a repo-wide rename of these. Changing here and opening a PR for this as well